search-app/src/cmr/search/services/query_execution/facets/granule_v2_facets.clj

```diff
@@ -0,0 +1,22 @@
+(ns cmr.search.services.query-execution.facets.granule-v2-facets
+  "Functions for generating v2 granule facets. Similar structure as v2 collection facets, but
+  granule fields. First major use case is supporting OPeNDAP virutal directories capability.")
+
+(def granule-facet-params->elastic-fields
+  {})
+
+(def granule-facet-fields
+  "Faceted fields for granules."
+  (keys granule-facet-params->elastic-fields))
+
+(def granule-fields->aggregation-fields
+  "Defines the mapping of granule parameter names to the aggregation parameter names."
+  {})
+
+(def granule-v2-facets-root
+  "Root element for the facet response"
+  {:title "Browse Granules"})
```

Comment:
New to facet code. Why is it that granule-v2-facets-root doesn't need a type while collection-v2-facets-root has it?  Is it because collection includes granules? Is there any relationship between granule facets and collection facets?

Reply:
Granules will eventually - this is just the first ticket that adds the minimal support.

https://bugs.earthdata.nasa.gov/browse/CMR-4684 is when it will get added in. The whole capability requires CMR-4680 through CMR-4687 to be implemented.